packages/contracts-watr/test/verifyDeployment/verifyDeployment.test.ts

```diff
@@ -0,0 +1,88 @@
+import { Contract, ethers } from 'ethers'
+import { JsonRpcProvider } from '@ethersproject/providers'
+import { expect } from 'chai'
+import { unknown as deployments } from '../../deployments-watr_local.json'
+import { TokenControllerV3__factory, TrueUSD__factory } from 'contracts'
+import { parseEther } from '@ethersproject/units'
+
+describe('verify deployment', () => {
+  const provider = new JsonRpcProvider('http://127.0.0.1:8822', 688)
+  const ownableInterface = [
```

Comment:
I wanted to suggest we use one of the Typechain-generated factories, but apparently, there are none like this. Which perhaps suggests we have a Solidity interface missing somewhere? 🤔

Reply:
Yes, I guess it can stay like this ✅